Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -132,6 +132,7 @@ HackStudioWidget::HackStudioWidget(const String& path_to_project)
     m_stop_action = create_stop_action();
     m_debug_action = create_debug_action();
     m_git_commit_action = create_git_commit_action();
+    m_git_commit_action->set_enabled(m_git_widget->initialize_if_needed(true));
```

Comment:
Can this not query `initialized()`?

Reply:
This is required because the repository won't be initialized by this time. The repository is only initialized when the git menu at the bottom is opened.

Reply:
No, the repository isn't initialized at this time, as it will only be initialized when the ``GitWidget`` is shown.